service/bandLevelService.ts

```diff
@@ -0,0 +1,13 @@
+import { BandLevel } from "../model/bandLevel";
+
+const axios = require("axios");
+
+module.exports.getBandLevels = async function (): Promise<BandLevel[]> {
+    try {
+        const response = await axios.get("http://localhost:8080/api/bandlevel-list");
```

Comment:
[Possible Future Improvement] - Using an environment variable is much safer then hardcoding "localhost".